.github/workflows/linux.yml

```diff
@@ -65,9 +65,10 @@ jobs:
         submodules: recursive
         fetch-tags: true
         persist-credentials: false
-    - uses: quansight-labs/setup-python@b9ab292c751a42bcd2bb465b7fa202ea2c3f5796 # v5.3.1
+    - uses: astral-sh/setup-uv@b5f58b2abc5763ade55e4e9d0fe52cd1ff7979ca
```

Comment:
Something is going wrong - the `3.13t` smoke test job builds for Python 3.12, not 3.13t